xamarin/Xamarin.Forms
# [Android|iOS] SwipeView in CollectionView selection

### Description of Change ###

Using SwipeView with CollectionView, we apply changes in this PR to allow the interaction with the CollectionView (for example, `SelectionChanged` event) when are not doing swipe.

![issue8774-ios](https://user-images.githubusercontent.com/6755973/70641480-fcea0780-1c3d-11ea-8cc6-87159a25fd61.gif)

_NOTE: Tests are still being done in the fix for Android. For that reason this branch has the "DO-NOT-MERGE!!!" tag._

### Issues Resolved ### 

- fixes #8774

### API Changes ###

 None

### Platforms Affected ### 

- iOS
- Android

### Behavioral/Visual Changes ###

None

### Before/After Screenshots ### 

Not applicable

### Testing Procedure ###
Launch Core Gallery and navigate to the issue 8774. Tap any item and verify that the `SelectionChanged` event of the CollectionView is executed.

### PR Checklist ###
